symphony-bdk-core/src/main/java/com/symphony/bdk/core/util/ConsumerWithThrowable.java

```diff
@@ -0,0 +1,12 @@
+package com.symphony.bdk.core.util;
```

Comment:
Let's move this class (and sisters) to package `com.symphony.bdk.core.util.function`, following [`java.util.function`](https://docs.oracle.com/javase/8/docs/api/java/util/function/package-summary.html#:~:text=Represents%20an%20operation%20that%20accepts%20two%20input%20arguments%20and%20returns%20no%20result.&text=Represents%20a%20function%20that%20accepts%20two%20arguments%20and%20produces%20a%20result.&text=Represents%20an%20operation%20upon%20two,same%20type%20as%20the%20operands.)